skyd/skyd.go

```diff
@@ -89,8 +89,8 @@ func (c *client) FileHealth(sp skymodules.SiaPath) (float64, error) {
 
 // Metadata returns the metadata of the skylink
 func (c *client) Metadata(skylink string) (skymodules.SkyfileMetadata, error) {
-	c.staticLogger.Trace("Entering Metadata.")
-	defer c.staticLogger.Trace("Exiting  Metadata.")
+	c.staticLogger.Trace("Entering Metadata")
```

Comment:
Not really. It tells me the execution path. Same as the siapath in the other discussion, I don't particularly care which branch goes where - I won't debug what has happened with a particular skylink. I want to see that the service goes through the right paths.

Also, the code is not really concurrent - we have one thread that's doing the scanning. We don't want it concurrent because we want the other servers to also have a chance at pinning some of the underpinned skylinks.